src/visualization/index.ts

```diff
@@ -0,0 +1,46 @@
+import {FunctionComponent, ComponentClass} from 'react'
+import {ViewProperties, Theme, TimeZone, TimeRange} from 'src/types'
+import {FromFluxResult} from '@influxdata/giraffe'
+
+export interface VisOptionProps {
+  properties: ViewProperties
+  results: FromFluxResult
+  update: (obj: any) => void
+}
+
+export interface VisProps {
+  properties: ViewProperties
+  result: FromFluxResult
+  theme?: Theme
+  timeZone?: TimeZone
+  timeRange?: TimeRange
+}
+
+export interface VisTypeRegistration {
```

Comment:
```suggestion
export interface Visualization {
```

this is the guy. it's what represents a visualization in our system. call it out loudly